src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/Worker.java

```diff
@@ -561,12 +578,14 @@ public Worker(
         this.leaderDecider = leaderDecider;
         this.shardSyncStrategy = createShardSyncStrategy(config.getShardSyncStrategyType());
         LOG.info(String.format("Shard sync strategy determined as %s.", shardSyncStrategy.getStrategyType().toString()));
+        this.leaderElectedPeriodicShardSyncManager = ObjectUtils.firstNonNull(periodicShardSyncManager,
```

Comment:
I think it's easier to read this if we just use Optional.orElseGet()

Reply:
See below